docs/models.md

```diff
@@ -0,0 +1,511 @@
+# Models
+
+## actor_states
+
+Actor states that were changed at an epoch. Associates actors states as single-level trees with CIDs pointing to complete state tree with the root CID (head) for that actor's state.
+
+| Column | Description                                      |
+| ------ | ------------------------------------------------ |
+| height | Epoch when this state change happened.           |
+| head   | CID of the root of the state tree for the actor. |
+| code   | CID identifier for the type of the actor.        |
+| state  | Top level of state data.                         |
```

Comment:
We have this data as COMMENT entries in the DB. How do we make sure that it does not get out of sync?